extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/OpenshiftApplyReplicasDecorator.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.kubernetes.deployment;
+
+import io.dekorate.kubernetes.decorator.Decorator;
+import io.dekorate.kubernetes.decorator.NamedResourceDecorator;
+import io.dekorate.openshift.decorator.ApplyReplicasDecorator;
+
+public class OpenshiftApplyReplicasDecorator extends ApplyReplicasDecorator {
+
+    public OpenshiftApplyReplicasDecorator(int replicas) {
+        super(replicas);
+    }
+
+    @Override
+    public Class<? extends Decorator>[] after() {
+        return new Class[] { NamedResourceDecorator.class };
+    }
+
+}
```

Comment:
@iocanel what do you think about this? Would you have a different suggestion on how to override after() method

Reply:
I would prefer if it also included `ApplyReplicasDecorator` (same reasoning as the kubernetes variant).